http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -30,15 +30,13 @@ sealed trait ContainerRegistry extends Product with Serializable {
 }
 object ContainerRegistry {
   final case object GCR extends ContainerRegistry {
-    def regex: Regex = """(.*gcr.io)\/(.+)\/(.+)(\:?.*)$""".r
-
+    val regex: Regex = """^(.*gcr.io)/([\w.-]+/[\w.-]+)(?::(\w[\w.-]+))?(?:@([\w+.-]+:[A-Fa-f0-9]{32,}))?$""".r
```

Comment:
Tweaked this regex to be more like the GCR one

Reply:
Yeah that's reasonable, although I think the suggestion is not quite right (there's an unmatched paren). I'll try this and improve the regex unit tests.

Reply:
Ah yea...I've updated it. Hope it works now.

Reply:
It does! added in latest commit